demonstrations/tutorial_qchem_external.py

```diff
@@ -0,0 +1,152 @@
+r"""
+
+Using PennyLane with PySCF and OpenFermion
+==========================================
+
+.. meta::
+    :property="og:description": Learn how to integrate external quantum chemistry libraries with PennyLane.
+    :property="og:image": https://pennylane.ai/qml/_images/external_libs.png
+
+.. related::
+    tutorial_quantum_chemistry Quantum chemistry with PennyLane
+    tutorial_vqe A brief overview of VQE
+    tutorial_givens_rotations Givens rotations for quantum chemistry
+    tutorial_adaptive_circuits Adaptive circuits for quantum chemistry
+
+*Author: Soran Jahangiri. Posted: 6 October 2022. Last updated: 6 October 2022*
+
+The quantum chemistry module in PennyLane, :mod:`qml.qchem  <pennylane.qchem>`, provides built-in
+methods for computing molecular integrals, solving Hartree-Fock equations and constructing
+`fully-differentiable <https://pennylane.ai/qml/demos/tutorial_differentiable_HF.html>`_ molecular
+Hamiltonians. PennyLane also allows using external libraries to perform these tasks and integrate
+the results within a desired workflow. In this tutorial, you will learn how to integrate PennyLane
+with the electronic structure package `PySCF <https://github.com/sunqm/pyscf>`_ and
+`OpenFermion <https://github.com/quantumlib/OpenFermion>`_ tools to compute molecular integrals and
+construct molecular Hamiltonians.
+
+Building molecular Hamiltonians
+-------------------------------
+In PennyLane, molecular Hamiltonians are built with the

```

Comment:
```suggestion
In PennyLane, Hamiltonians for quantum chemistry are built with the
```
to avoid repetition of molecular Hamiltonian